adaptive_scheduler/monitoring/opensearch_seeing.py

```diff
@@ -0,0 +1,90 @@
+from opensearchpy import OpenSearch
+from dateutil.parser import parse
+from datetime import datetime, timedelta
+
+
+class ConnectionError(Exception):
+    pass
+
+
+def get_seeing(resources, opensearch_url, os_index, os_excluded_observatories, lookback_minutes):
+    ''' Get seeing value from an opensearch index structured like the banzai qc index. Any opensearch index which has keys on telescope
+        , site, observatory, and seeing value should work though.
+    '''
+    os = OpenSearch(opensearch_url, http_compress=True)
+
+    seeing_by_resource = {}
+    for resource in resources:
+        # Resource format is from configdb telescope data, so telescope.enclosure.site
+        telescope, enclosure, site = resource.split('.')
+        if enclosure not in os_excluded_observatories:
+            seeing_query = _get_seeing_query(site, enclosure, telescope, lookback_minutes)
+            try:
```

Comment:
Eh kinda minor but hard-coding it to 2 times with no backoff or anything feels a little...specific? Feels like we could use an actual retry decorator here to do it properly - do we use retry or tenacity in this project?

Reply:
okay, added retry library but left it at 2 retries... I don't want it to block operation of the scheduler with too many exponential retries.